resources/definitions/dagoma_neva.def.json

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "Dagoma_neva",
+    "id": "dagoma_neva",
```

Comment:
I think those `id`s are not necessary since the id is taken from the filename.

Reply:
True. Ruben mentioned it for the Disco Easy one but I forgot to check the others for useless IDs.

Reply:
Fixed in ed1e47d